src/plugins/console/public/application/containers/main/i18n.ts

```diff
@@ -12,8 +12,8 @@ export const MAIN_PANEL_LABELS = {
   consolePageHeading: i18n.translate('console.pageHeading', {
     defaultMessage: 'Console',
   }),
-  importExportButton: i18n.translate('console.importExportButtonLabel', {
-    defaultMessage: 'Import/Export',
+  exportFileButton: i18n.translate('console.exportFileButton', {
+    defaultMessage: 'Export as file',
```

Comment:
This im still not sure, I left the one in my PR as just `import` as I thought seems simpler and shorter.. but perhaps we can get some input from @florent-leborgne on what should be best?

Reply:
From what I'm seeing there are 2 things I'd consider:
- This button is to export the requests (I don't think it's that obvious given the location of the button). Or does it also export the response when there's one?
- I don't think "a file" is needed because "Export" only usually conveys that idea. That said, is there something else users could expect, like exporting a request into a different language maybe, and be surprised that they get a file?? 

Based on that (sorry if I'm missing other elements or context), I think it could be `Export requests`.

Just in case, we could use a tooltip with a longer label when hovering the button to say "Export all console requests to a TXT file"

Reply:
To answer the first questions:

* This is **only** meant to be used for exporting the requests in the editor as they are into a file (so that the user can keep them separately or to share with someone else)
* To convert requests to another language its another flow entirely (through context menu) so the idea of this particular button is to just export whatever is in the editor in a file.

We are also going to have an `import` option next to it, not sure if adding the `requests` copy also might make it a bit too verbose:

![Screenshot 2024-09-04 at 13 22 15](https://github.com/user-attachments/assets/0ded65d6-6355-48af-9f71-a2ddf536a569)
![Screenshot 2024-09-04 at 13 22 03](https://github.com/user-attachments/assets/01ddcb0d-cb58-4ca8-8548-15cac6c67ad2)

I do think the tooltip might be a good idea just in case to clarify what will be happening

Reply:
👍 Sounds good.

I do prefer the one with "requests" as I think it's much clearer, especially when there's a response showing just under the buttons, but I'll leave it up to you

Reply:
Thanks for the feedback @sabarasaba and @florent-leborgne! I also like the version with "requests" more. Updated the label and added a tooltip with the text that @florent-leborgne suggested with the latest changes (PR screenshot updated).